examples/animation/animation_graph.rs

```diff
@@ -0,0 +1,538 @@
+            color: Srgba(WHITE),
+            brightness: 100.0,
+        })
+        .run();
+}
+
+fn setup(
+    mut commands: Commands,
+    mut asset_server: ResMut<AssetServer>,
+    mut animation_graphs: ResMut<Assets<AnimationGraph>>,
+) {
+    // Create the assets.
+    setup_assets(&mut commands, &mut asset_server, &mut animation_graphs);
+
+    // Create the scene.
+    setup_scene(&mut commands, &mut asset_server);
+    setup_camera_and_light(&mut commands);
+
+    // Create the UI.
+    setup_help_text(&mut commands, &mut asset_server);
+    setup_node_rects(&mut commands, &mut asset_server);
+    setup_node_lines(&mut commands);
+}
+
+/// Creates the assets, including the animation graph.
+fn setup_assets(
+    commands: &mut Commands,
+    asset_server: &mut AssetServer,
+    animation_graphs: &mut Assets<AnimationGraph>,
+) {
+    // Create the nodes.
+    let mut animation_graph = AnimationGraph::new();
+    let blend_node = animation_graph.add_blend(0.5, animation_graph.root);
+    let node_indices: [_; 3] = [
+        animation_graph.add_clip(
+            asset_server.load("models/animated/Fox.glb#Animation0"),
+            1.0,
+            animation_graph.root,
+        ),
+        animation_graph.add_clip(
+            asset_server.load("models/animated/Fox.glb#Animation1"),
+            1.0,
+            blend_node,
+        ),
+        animation_graph.add_clip(
+            asset_server.load("models/animated/Fox.glb#Animation2"),
+            1.0,
+            blend_node,
+        ),
+    ];
+
+    // Add the graph.
+    let animation_graph = animation_graphs.add(animation_graph);
+
+    // Save the assets in a resource.
+    commands.insert_resource(AppAssets {
+        animation_graph,
+        node_indices,
+    });
+}
+
+/// Spawns the animated fox.
+fn setup_scene(commands: &mut Commands, asset_server: &mut AssetServer) {
+    commands.spawn(SceneBundle {
+        scene: asset_server.load("models/animated/Fox.glb#Scene0"),
+        transform: Transform::from_scale(Vec3::splat(0.05)),
+        ..default()
+    });
+}
+
+/// Spawns the camera and point light.
+fn setup_camera_and_light(commands: &mut Commands) {
+    commands.spawn(Camera3dBundle {
+        transform: Transform::from_xyz(-10.0, 5.0, 13.0).looking_at(Vec3::ZERO, Vec3::Y),
+        ..default()
+    });
+
+    commands.spawn(PointLightBundle {
+        point_light: PointLight {
+            intensity: 10000000.0,
+            shadows_enabled: true,
+            ..default()
+        },
+        transform: Transform::from_xyz(-4.0, 6.0, 13.0),
+        ..default()
+    });
+}
+
+/// Places the help text at the top left of the window.
+fn setup_help_text(commands: &mut Commands, asset_server: &mut AssetServer) {
+    commands.spawn(TextBundle {
+        text: Text::from_section(
+            HELP_TEXT,
+            TextStyle {
+                font: asset_server.load("fonts/FiraMono-Medium.ttf"),
+                font_size: 24.0,
```

Comment:
There has been a (somewhat lazy lately) effort to standardize "top left instructions" in examples with:

- White (default) text color
- Default font
- 20px font size
- 12px from top/left.

See <https://github.com/bevyengine/bevy/pull/8478>.

Reply:
Done